vendor/k8s.io/kubernetes/pkg/client/leaderelection/leaderelection.go

```diff
@@ -214,7 +214,7 @@ func (le *LeaderElector) acquire() {
 		le.config.EventRecorder.Eventf(&api.Endpoints{ObjectMeta: le.config.EndpointsMeta}, api.EventTypeNormal, "%v became leader", le.config.Identity)
 		glog.Infof("sucessfully acquired lease %v/%v", le.config.EndpointsMeta.Namespace, le.config.EndpointsMeta.Name)
 		close(stop)
-	}, 0, stop)
+	}, time.Millisecond, stop)
```

Comment:
If we still need this change, it should be as part of the coreos/kubernetes fork, then we should be vendoring that.